src/lib/viewers/BaseViewer.js

```diff
@@ -1000,6 +1006,30 @@ class BaseViewer extends EventEmitter {
         return AnnotationMode.NONE;
     }
 
+    /**
+     * Updates localStorage with the state of the ftux cursor toggle
+     *
+     * @protected
+     * @param {string} key
+     * @return {void}
+     */
+    handleFtuxCursorToggle(key) {
+        if (!this.cache.get(key)) {
+            this.cache.set(key, true, true);
```

Comment:
Also, should we perform this ftux on a per user id basis?

Reply:
After doing some digging, it looks like we don't need to do it on a per user id basis because localStorage sessions are different for unique users.